### PR TITLE
Introduce Spring Actuator

### LINKED ISSUE #288: Introduce Spring Boot Actuator
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html

We need to care password field for `/actuator/env`. 
https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html#management.endpoint.env.keys-to-sanitize

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #288

### CODE REVIEW

File: src/main/resources/config/application.yml [hunk 1]
```diff
@@ -15,6 +15,24 @@ spring:
...
+      percentiles:
+        http.server.requests: 0.5, 0.75, 0.95, 0.99
+  endpoint:
+    metrics.enabled: true
+    prometheus.enabled: true
+    heapdump.enabled: false
+    health:
+      show-details: always
+    env:
+      keys-to-sanitize: .*password.*
```
Comment (g-yuqing) [inline src/main/resources/config/application.yml:32]: Why do we need to customize this setting?

File: src/main/resources/config/application.yml [hunk 2]
```diff
@@ -15,6 +15,24 @@ spring:
   resources:
     static-locations: file:web
 
+# metrics
```
Comment (g-yuqing) [inline src/main/resources/config/application.yml:18]: nit) # Metrics to keep consistency

**Review by g-yuqing — APPROVED** [review]

File: src/main/resources/config/application.yml (hunk 1 above)
Comment (ebyhr) [reply src/main/resources/config/application.yml:32]: Yanagishima config has `hive.jdbc.password.{datasource}` property, but default `keys-to-sanitize` expects "ending" with `password`. We can see raw password in `/actuator/env` endpoint without this customization. 
https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/howto-actuator.html#howto-sanitize-sensible-values

**Comment by ebyhr** [comment]
Merged, thanks for your review!

**Pull request merged**